modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/graphql/v1_0/test/ObjectDefinitionGraphQLTest.java

```diff
@@ -329,6 +331,101 @@ public void testAddObjectEntry() throws Exception {
 			JSONCompareMode.STRICT);
 	}
 
+	@Test
+	@TestInfo("LPD-49283")
+	public void testAddObjectEntryWithMultiselectPicklistField()
+		throws Exception {
+
+		ListTypeDefinition listTypeDefinition =
+			ListTypeDefinitionLocalServiceUtil.addListTypeDefinition(
+				null, TestPropsValues.getUserId(),
+				LocalizedMapUtil.getLocalizedMap(StringUtil.randomId()), false,
+				Collections.emptyList());
+
+		String listFieldValueKey1 = StringUtil.randomId();
+		String listFieldValueKey2 = StringUtil.randomId();
+		String listFieldValueKey3 = StringUtil.randomId();
+
+		_addListTypeEntry(listTypeDefinition, listFieldValueKey1);
+		_addListTypeEntry(listTypeDefinition, listFieldValueKey2);
+		_addListTypeEntry(listTypeDefinition, listFieldValueKey3);
+
+		String picklistFieldName = StringUtil.randomId();
```

Comment:
This is not sorted.

multiselectPicklistFieldName > picklistFieldName

please update everything here to have the multiSelect picklist before the picklist references